app/grandchallenge/components/views.py

```diff
@@ -126,15 +126,15 @@ class MultipleCIVProcessingBaseView(
     raise_exception = True
     success_message = None
 
-    def process_data_for_object(self, data):
-        raise NotImplementedError
-
     @property
     def base_object(self):
         raise NotImplementedError
 
+    def process_data(self, data):
+        raise NotImplementedError

```

Comment:
I don't know if I'm looking at an outdated version again, but I don't see his being implemented on any of the views that subclass this base view, and it's also not used further, so this should go.